services/streammanager/wunderkind/wunderkindmanager.go

```diff
@@ -0,0 +1,110 @@
+package wunderkind
+
+import (
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/service/lambda"
+	jsoniter "github.com/json-iterator/go"
+
+	"github.com/rudderlabs/rudder-go-kit/awsutil"
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	backendconfig "github.com/rudderlabs/rudder-server/backend-config"
+	"github.com/rudderlabs/rudder-server/cmd/rudder-cli/config"
+	"github.com/rudderlabs/rudder-server/services/streammanager/common"
+)
+
+const (
+	INVOCATION_TYPE         = "RequestResponse"
+	WUNDERKIND_REGION       = "WUNDERKIND_REGION"
+	WUNDERKIND_IAM_ROLE_ARN = "WUNDERKIND_IAM_ROLE_ARN"
+	WUNDERKIND_EXTERNAL_ID  = "WUNDERKIND_EXTERNAL_ID"
+	WUNDERKIND_LAMBDA       = "WUNDERKIND_LAMBDA"
```

Comment:
Use camel case instead of snake case 
```suggestion
	InvocationType       = "RequestResponse"
	WunderkindRegion     = "WUNDERKIND_REGION"
	WunderkindIamRoleArn = "WUNDERKIND_IAM_ROLE_ARN"
	WunderkindExternalId = "WUNDERKIND_EXTERNAL_ID"
	WunderkindLambda     = "WUNDERKIND_LAMBDA"
```

Reply:
addressed